code/graphics/gropengl.cpp

```diff
@@ -43,7 +43,7 @@
 #include <glad/glad.h>
 
 // minimum GL version we can reliably support is 2.0
-static const int MIN_REQUIRED_GL_VERSION = 12;
+static const int MIN_REQUIRED_GL_VERSION = 20;
 
 // minimum GLSL version we can reliably support is 110
 static const int MIN_REQUIRED_GLSL_VERSION = 110;
```

Comment:
Yes, these are the matching versions of OpenGL and GLSL. You can see the table here: https://www.opengl.org/wiki/Core_Language_(GLSL)#OpenGL_and_GLSL_versions

It was not until OpenGL 3.3 and above, that the corresponding GLSL version matches the OpenGL version. So OpenGL 4.1 uses GLSL 4.10.